doc/api/repl.md

```diff
@@ -406,6 +406,8 @@ changes:
   * `useGlobal` {boolean} If `true`, specifies that the default evaluation
      function will use the JavaScript `global` as the context as opposed to
      creating a new separate context for the REPL instance. Defaults to `false`.
```

Comment:
I think the key point is to make readers notice the default value is `false`, while node CLI REPL use `true` in comparison.
Your suggestion is good. I try to make it less wordy.